lacework/assets/logos/README.md

```diff
@@ -0,0 +1,19 @@
+# Logos
```

Comment:
This is not needed anymore. You should request design to create and deploy the logos